go.mod

```diff
@@ -253,7 +254,11 @@ require (
 
 replace (
 	github.com/google/cel-go => github.com/google/cel-go v0.17.7
+	github.com/openshift/hypershift/api => ./api
 	k8s.io/kubernetes => k8s.io/kubernetes v0.29.0
 	kubevirt.io/client-go => kubevirt.io/client-go v0.0.0-00010101000000-000000000000
 	kubevirt.io/containerized-data-importer-api => github.com/kubevirt/containerized-data-importer-api v1.41.1-0.20211201033752-05520fb9f18d
+	sigs.k8s.io/apiserver-network-proxy/konnectivity-client => sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.28.1
+	sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.5.3
```

Comment:
Why are we pinning these two deps, L262 & L263? We were trying to get away from that as much as possible I thought.

Reply:
During the progression of this issue, I've been seeing that both capi deps were bumping by `make verify` command (dunno why tbh). I think I can unpin them now